python/lsst/sims/photUtils/Sed.py

```diff
@@ -481,6 +481,58 @@ def setFlatSED(self, wavelen_min=None,
         self.name = name
         return
 
+    def read_close_SED(self, teff, feH, logg):
+        """
+        Check the cached Kuruz models and load the model closest to the input stellar parameters
```

Comment:
The docstring should mention that matching Teff is prioritized over matching feH is prioritized over matching logg.  I don't believe our grid of Kurucz models is uniform, so the order in which you match parameters may be relevant.